docs/release-notes/8.12.asciidoc

```diff
@@ -5,6 +5,22 @@
 [[release-notes-8.12.2]]
 === 8.12.2
 
+[discrete]
+[[known-issue-8.12.2]]
+==== Known issues
+// tag::known-issue-178207[]
+[discrete]
+.Rule filters can't be adjusted for certain rule types that are using data views  
```

Comment:
```suggestion
.Rule filters can't be adjusted for rules that are using data views  
```
Suggested update only because I don't think the rule _type_ is what matters, but more that the rule is using data views as it's source.

Reply:
Applying @e40pud's feedback from Slack:

 ```suggestion
.Rule filters can't be adjusted for custom query, indicator match, and new term rules using data views  
```